ascii.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+# Define your ASCII art with $ characters
+ascii_art = """
+  $$$$$$$                                                                                   
+  $$    $$                                                                                  
+  $$    $$   $$$$$$   $$    $$  $$$$$$$   $$$$$$$    $$$$$$    $$$$$$   $$$$$$$    $$$$$$   
+  $$$$$$$   $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  
+  $$    $$  $$        $$    $$  $$    $$  $$    $$  $$$$$$$$  $$        $$    $$  $$$$$$$$  
+  $$    $$  $$        $$    $$  $$    $$  $$    $$  $$        $$        $$    $$  $$        
+  $$$$$$$   $$         $$$$$$   $$    $$  $$    $$   $$$$$$$  $$        $$    $$   $$$$$$$
+"""
+
+# Replace $ with 0 or 1 randomly
```

Comment:
You're right, hiding a message here would be quite neat. Maybe randomly rotate (ROT-n) some secret message?

Reply:
Flag added 😎